orchestra/migrations/0049_auto_20160428_1422.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-28 14:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orchestra', '0048_merge'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='staffingrequest',
+            name='communication_method',
+            field=models.IntegerField(choices=[(0, 'slack'), (1, 'email')], default=0),
```

Comment:
we cant do it right away. Because the field requires a default value. So it's a hack around it